### DISCUSSION TITLE
Option to show the year similar to the TMDB rating

### DISCUSSION THREAD

**Discussion by Slashin8r** [Dreamlinerm/Netflix-Prime-Auto-Skip#125] [Ideas]
I suggest adding the year like how the TMDB rating is in the lower right corner. You should be able to grab the year from TMDB, same as the rating.

Services like Netflix and Prime do not show the year on their content until you click multiple times to bring up the individual details. All services add content regularly and the content they add is not always new. Seeing the year would instantly let users know if the newly added content is actually new.

An issue specific to Netflix: The wrong year is displayed in the details of a series on Netflix. They went against the majority and they show the year of the most recent season of the series. This is definitely wrong! An example why it is wrong would be Prison Break. This series ran 4 seasons starting in 2005 and ending in 2009, but then they came back for a 5th season in 2017. Netflix shows it as a 2017 series, when we all know this is a 2005 series! There are plenty of other examples as well.

**Comment by Dreamlinerm** [comment]
> I suggest adding the year like how the TMDB rating is in the lower right corner. You should be able to grab the year from TMDB, same as the rating.
> 
> Services like Netflix and Prime do not show the year on their content until you click multiple times to bring up the individual details. All services add content regularly and the content they add is not always new. Seeing the year would instantly let users know if the newly added content is actually new.
> 
> An issue specific to Netflix: The wrong year is displayed in the details of a series on Netflix. They went against the majority and they show the year of the most recent season of the series. This is definitely wrong! An example why it is wrong would be Prison Break. This series ran 4 seasons starting in 2005 and ending in 2009, but then they came back for a 5th season in 2017. Netflix shows it as a 2017 series, when we all know this is a 2005 series! There are plenty of other examples as well.

If it looks ok, i will show the year additionaly on the right corner. I will look at, if i can disable the hover animation if you hover on the rating. If i show the year i will show zhe year of the specific season. I will maybe make the year optional. Might program this after Christmas.

**Comment by Dreamlinerm** [comment]
> I suggest adding the year like how the TMDB rating is in the lower right corner. You should be able to grab the year from TMDB, same as the rating.
> 
> Services like Netflix and Prime do not show the year on their content until you click multiple times to bring up the individual details. All services add content regularly and the content they add is not always new. Seeing the year would instantly let users know if the newly added content is actually new.
> 
> An issue specific to Netflix: The wrong year is displayed in the details of a series on Netflix. They went against the majority and they show the year of the most recent season of the series. This is definitely wrong! An example why it is wrong would be Prison Break. This series ran 4 seasons starting in 2005 and ending in 2009, but then they came back for a 5th season in 2017. Netflix shows it as a 2017 series, when we all know this is a 2005 series! There are plenty of other examples as well.

I can add the last two digits of the year to the lower left corner, similar to the picture maybe?
![grafik](https://github.com/user-attachments/assets/c6a7f980-00a1-4eab-89a6-c80da06be681)

**Reply by Dreamlinerm** [reply]
or leave the full year in the lower right corner.
![grafik](https://github.com/user-attachments/assets/e62b5de9-a7ba-4d43-8221-9f71aa3eac0f)
I think i prefer this, since it looks better.

**Reply by Dreamlinerm** [reply]
I think im gonna make this  an optional setting

**Reply by Dreamlinerm** [reply]
I did the lower right version and made this an optional setting. I will upload the version later maybe monday.

**Comment by Dreamlinerm** [comment]
@Slashin8r I added the option in the new version. I did this so late since I wanted to add other stuff in the same version which did not pan out.
